contracts/TokenDistributor.sol

```diff
@@ -0,0 +1,193 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Just realized we use `// SPDX-License-Identifier: GPL-3.0-or-later` in margin-manager. @JChiaramonte7 not sure if this matters or not

Reply:
Yes good catch, we're moving to `// SPDX-License-Identifier: GPL-3.0-or-later` moving forward